go.mod

```diff
@@ -322,6 +323,7 @@ require (
 )
 
 replace (
+	github.com/celestiaorg/utils => github.com/celestiaorg/utils v0.0.0-20230111201732-2d8b5f8774ad // uses an unmerged PR, delete when celestiaorg/utils PR#5 is merged
```

Comment:
Can be removed 
Ref: https://github.com/celestiaorg/utils/releases/tag/v0.1.0

Reply:
👍🏻